docs/proofs/output_lfsc.rst

```diff
@@ -0,0 +1,14 @@
+Proof format: LFSC
+==================
+
+Using the flag :ref:`proof-format-mode=lfsc <lbl-option-proof-format-mode>`, cvc5 outputs proofs in the LFSC proof format.
+
+The LFSC proof format is based on the LF logical framework extended with computational side conditions, as described `this journal article <https://homepage.divms.uiowa.edu/~ajreynol/fmsd12.pdf>`. A high performance C++ proof checker for LFSC is available `here <https://github.com/cvc5/LFSC>`.
```

Comment:
```suggestion
The LFSC proof format is based on the LF logical framework extended with computational side conditions, as described `this journal article <https://homepage.divms.uiowa.edu/~ajreynol/fmsd12.pdf>`. A high performance C++ proof checker for LFSC is available `here <https://github.com/cvc5/LFSC>`_.
```

Reply:
for external links:

    `name <url>`_

https://www.sphinx-doc.org/en/master/usage/restructuredtext/basics.html#hyperlinks